tests/unit/lms/validation/blackboard_test.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+
+from lms.validation._exceptions import ValidationError
+from lms.validation.blackboard import BlackboardListFilesSchema
+from tests import factories
+
+
+class TestBlackboardListFilesSchema:
+    @pytest.mark.parametrize(
+        "input_,output",
+        [
+            (  # The input is a dict with a "results" key whose value is the list
+                # of files.
+                {
+                    "results": [
+                        {
+                            "id": "_7851_1",
+                            "name": "File_1.pdf",
+                            "modified": "2021-06-02T15:39:51.000Z",
+                            "unknown_field": "this_should_be_included",
+                        },
+                        {
+                            "id": "_7851_2",
+                            "name": "File_2.pdf",
+                            "modified": "2021-05-02T15:39:51.000Z",
+                        },
+                        {
+                            "id": "_7851_3",
+                            "name": "File_3.pdf",
+                            "modified": "2021-04-02T15:39:51.000Z",
+                        },
+                    ],
+                },
+                # The output is just the list of files, directly.
+                [
+                    {
+                        "id": "_7851_1",
+                        "name": "File_1.pdf",
+                        "modified": "2021-06-02T15:39:51.000Z",
+                        "unknown_field": "this_should_be_included",
+                    },
+                    {
+                        "id": "_7851_2",
+                        "name": "File_2.pdf",
+                        "modified": "2021-05-02T15:39:51.000Z",
+                    },
+                    {
+                        "id": "_7851_3",
+                        "name": "File_3.pdf",
+                        "modified": "2021-04-02T15:39:51.000Z",
+                    },
+                ],
+            )
```

Comment:
Could use a test for the valid case when the list of files is empty.

Once the schema is extended to actually mutate the data (excluding unknown fields, changing the names of some fields, ...) this test will have to be improved a bit

Reply:
I assume you'll probably end up with some extra versions of the test (like empty lists), but if you don't the mark can probably go.